src/atoms/DymoHeading.js

```diff
@@ -1,22 +1,13 @@
 import React from 'react';
 import propTypes from 'prop-types';
-
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 
 import { Heading } from '..';
 
 const FormattedHeading = styled(Heading)`
   display: inline-block;
   margin: 0 0 calc(2/3*${props => props.theme.variables.verticalBase});
   ${props => (props.hasImage ? 'max-width: 90%;' : `max-width: calc(100% - ${props.theme.variables.horizontalBase});`)}
-
-	@media (min-width: ${props => props.theme.flexboxgrid.breakpoints.sm}em) {
-		${props => ((props.flexingSize === 'full') ? css`
```

Comment:
flexingSize is not used anymore. Should remove from props and default value

Reply:
Done, good catch